plugins/logitech-scribe/logitech-scribe.quirk

```diff
@@ -0,0 +1,5 @@
+[VIDEO4LINUX\VEN_046D&DEV_08E2]
+Plugin = logitech_scribe
+InstallDuration = 120
+Flags = is-scribe
```

Comment:
Is this really valuable?

Reply:
Removed. Had it for future hardware models